README.md

```diff
@@ -1 +1,3 @@
-# repo
\ No newline at end of file
+# repo
+
+Some changes here
```

Comment:
@project-health1 what do think of this change?

Reply:
I'm not sure.